dubbo-monitor/dubbo-monitor-default/src/main/java/org/apache/dubbo/monitor/dubbo/MetricsFilter.java

```diff
@@ -237,7 +237,7 @@ public Result invoke(Invocation invocation) throws RpcException {
 
                 List res = collector.build();
                 res.addAll(getThreadPoolMessage());
-                return AsyncRpcResult.newDefaultAsyncResult(JSON.toJSONString(res), invocation);
+                return AsyncRpcResult.newDefaultAsyncResult(new Gson().toJson(res), invocation);
```

Comment:
Was it necessary to make `Gson` object singleton in `dubbo-common` module?

Reply:
That's good advic!